tests/let/timportc.nim

```diff
@@ -0,0 +1,16 @@
+discard """
+targets: "c cpp js"
+"""
+
+when defined(c) or defined(cpp):
+  {.emit:"""
+  const int TEST = 123;
+  """.}
+
+when defined(js):
+  {.emit:"""
+  const TEST = 123;
+  """.}
+
+let TEST {.importc, nodecl.}: cint
```

Comment:
2 birds one stone:
can you add a test for the macro I mentioned in https://github.com/nim-lang/Nim/pull/14258/files#r423781254 as well as making sure a mixed let section works, eg:

```nim
let
  TEST0 = 1
  TEST1 {.importc, nodecl.}: cint
  TEST2 {.importc, nodecl.}: cint
```

Reply:
Well this works by how the checking is performed (it loops over all the statements and this check is inside that loop), so I don't really see what this would be good for. And now we don't test that it works _without_ another let statement..